configure.ac

```diff
@@ -352,6 +352,33 @@ else
     AC_MSG_RESULT([yes])
 fi
 
+##########################
+# KVM vmapoll workaround
```

Comment:
@igor-ivanov - no runtime check at fast path:
* on init code checks if KVM or not and set appropriate func ptr
* in the fast path - always use fptr